contrib/Zipkin/SpanConverter.php

```diff
@@ -51,13 +53,14 @@ public function convert(Span $span)
             ],
             'name' => $span->getSpanName(),
             'timestamp' => (int) ($span->getStartEpochTimestamp() / 1e3), // RealtimeClock in microseconds
-            'duration' => (int) (($span->getEnd() - $span->getStart()) / 1e3), // Diff in microseconds
+            'duration' => (int) (($span->getEnd() - $span->getStart()) / 1e3), // Diff in microseconds,
```

Comment:
Was this trailing comma added intentionally?

Reply:
no. I will remove it